.github/workflows/push-misc.yml

```diff
@@ -0,0 +1,12 @@
+name: Push to any branch
+on:
+  push:
+    branches-ignore:
+      - main
+    tags-ignore:
+      - '*'
```

Comment:
This should probably not be on push to branches, but on pull requests?

Reply:
hrm I guess that's a better approach for an open project, yeah.

Reply:
Done! :3